platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/binding/soap/SoapValidator.java

```diff
@@ -13,36 +13,14 @@
  */
 package org.codice.ddf.security.idp.binding.soap;
 
-import ddf.security.samlp.SimpleSign;
 import ddf.security.samlp.SystemCrypto;
-import ddf.security.samlp.ValidationException;
 import ddf.security.samlp.impl.EntityInformation;
 import java.util.Map;
 import org.codice.ddf.security.idp.binding.api.Validator;
 import org.codice.ddf.security.idp.binding.api.impl.ValidatorImpl;
-import org.opensaml.saml.saml2.core.AuthnRequest;
 
 public class SoapValidator extends ValidatorImpl implements Validator {
   public SoapValidator(SystemCrypto systemCrypto, Map<String, EntityInformation> serviceProviders) {
     super(systemCrypto, serviceProviders);
   }
-
```

Comment:
These methods are taken out because they override their parents only to call `super`.